client/src/plus/deep-dives/sample-articles.tsx

```diff
@@ -238,6 +240,13 @@ export function PlanningForBrowserSupport() {
         many are on mobile devices, and which versions of evergreen browsers
         that are in use.
       </p>
+      <figure>
+        <img src="" width="" height="" alt="" />
```

Comment:
Here's the image: https://drive.google.com/file/d/12DihQAsu-_nBaTQRUlwGWD_RfEGnTQ9d/view?usp=sharing